source/Assets/Plugins/Didomi/Tests/SyncUserTestsSuite.cs

```diff
@@ -0,0 +1,185 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using IO.Didomi.SDK;
+using IO.Didomi.SDK.Events;
+
+/// <summary>
+/// Tests related to sharing consent with Webview / Web SDK
+/// </summary>
+public class SyncUserTestSuite : DidomiBaseTests
+{
+    private const string testUserId = "d13e49f6255c8729cbb201310f49d70d65f365415a67f034b567b7eac962b944eda131376594ef5e23b025fada4e4259e953ceb45ea57a2ced7872c567e6d1fae8dcc3a9772ead783d8513032e77d3fd";
```

Comment:
if this is an encrypted ID I'd suggest to name it something like `encryptedTestUserId` to make it explicit

Reply:
I don't believe this one is encrypted. It is the same user id we use in other platforms sync tests